scene/main/node.cpp

```diff
@@ -1685,13 +1685,13 @@ Node *Node::get_child(int p_index, bool p_include_internal) const {
 		if (p_index < 0) {
 			p_index += data.children_cache.size();
 		}
-		ERR_FAIL_INDEX_V(p_index, (int)data.children_cache.size(), nullptr);
+		ERR_FAIL_INDEX_V_MSG(p_index, (int)data.children_cache.size(), nullptr, vformat("Invalid child index: %d.", p_index));
```

Comment:
Taken slightly modified from `move_child`, both could be improved but unsure what to add further

Reply:
Perhaps `Failed to get_child() at index %d. The node (preferably a nodename if available), only has `data.children_cache.size()` children.